tests/arns.test.mjs

```diff
@@ -1509,4 +1513,225 @@ describe('ArNS', async () => {
   });
 
   // TODO: add several error scenarios
+
+  describe('ArNS Discount Eligible Gateways', () => {
+    const joinedGateway = 'joined-unique-gateway-'.padEnd(43, '0');
+    const firstEpochTimestamp = 1719900000000;
+    const afterDistributionTimestamp =
+      firstEpochTimestamp + 1000 * 60 * 60 * 24 + 1000 * 60 * 40;
+
+    let arnsDiscountMemory;
+    before(async () => {
+      // create gateways
+      const { memory: join1Memory } = await joinNetwork({
+        address: joinedGateway,
+        quantity: 200_000_000_000,
```

Comment:
make sure this timestamp equal to or less than the firstEpochTimestamp

Reply:
3c6a2dc

Reply:
had to add a followup to get this to pass: 32cd72e